solidity/contracts/converter/SwapSettings.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity 0.4.26;
+
+import "../utility/Owned.sol";
+
+contract SwapSettings is Owned {
+  uint32 internal constant PROTOCOL_FEE_RESOLUTION = 1000000;
+  address public wrbtcAddress;
+	address public sovTokenAddress;
+	address public feesController;
+  uint32 public protocolFee;
+
+  /**
+	 * @dev triggered when the protocol fee sorage is set/updated
+	 *
+	 * @param _prevProtocolFee previous protocol fee percentage
+	 * @param _newProtocolFee new protocol fee percentage
+	 */
+	event ProtocolFeeUpdate(uint32 _prevProtocolFee, uint32 _newProtocolFee);
```

Comment:
for some reason doesn't include `address indexed sender`. just inconsistent behavior.